### PR TITLE
Update the library to be Dart2 compatible.

### PR DESCRIPTION
**Pull request by matanlurey**
/cc @isoos @kevmoo 

... this is required to unblock the AngularDart build.

### CODE REVIEW

**Comment by kevmoo** [comment]
@matanlurey – did you verify it fixes things?

File: pubspec.yaml [hunk 1]
```diff
@@ -6,7 +6,8 @@ description: JavaScript bindings for the Service Worker API.
 homepage: https://github.com/isoos/service_worker
```
Comment (kevmoo) [inline pubspec.yaml]: Also bump the version?

**Comment by matanlurey** [comment]
@kevmoo I verified the analyzer doesn't crash.

**Comment by kevmoo** [comment]
...and that the HN PWA still works? :-)

**Comment by matanlurey** [comment]
No, too much work.

**Comment by isoos** [comment]
I've published 0.2.0

**Comment by kevmoo** [comment]
Thanks!

**Pull request merged**